app/controllers/inquiries_controller.rb

```diff
@@ -0,0 +1,20 @@
+class InquiriesController < ApplicationController
+  def create
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for create is too high. [16.06/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)

<!-- Comment by [SideCI](https://sideci.com) -->